### DISCUSSION TITLE
Nui input highlights

### DISCUSSION THREAD

**Discussion by GiuseppeMP** [folke/noice.nvim#612] [Q&A]
Hey guys, I'm using transparent config, everything is fine but in NvimTree:

![image](https://github.com/folke/noice.nvim/assets/4944009/bace5b91-d079-4eb0-986a-221f674e9160)

Can anyone tell me what the highlight is here and where this black background is coming from?

![image](https://github.com/folke/noice.nvim/assets/4944009/babca1e4-83e6-4a0a-be89-0aaf0f6571f4)

[My config.](https://github.com/GiuseppeMP/my-nvim-config/blob/f853778ff5fc3685e2b550caec78c1417d57017d/lua/user/plugins/configs/noice.lua#L34)

Thx!

**Comment by folke** [comment]
that's not Noice? But nvimtree?

**Reply by GiuseppeMP** [reply]
it was dressing:

```lua
require('dressing').setup({
    input = {
        win_options = {
            winblend = 0,
        },
    },
})
```

fixed!
